cumulus/scripts/migrate_storage_to_genesis/identityMigration.js

```diff
@@ -0,0 +1,108 @@
+const { u8aToHex, hexToString } = require('@polkadot/util');
+const { xxhashAsHex } = require('@polkadot/util-crypto');
+
+const migrateIdentity = async (key, data, api) => {
+	// Register the new `Registration` format type
+	api.registry.register({
+		IdentityInfoNew: {
+		_fallback: 'IdentityInfoTo198',
+		display: 'Data',
+		legal: 'Data',
+		web: 'Data',
+		matrix: 'Data',
+		email: 'Data',
+		pgpFingerprint: 'Option<H160>',
+		image: 'Data',
+		twitter: 'Data',
+		github: 'Data',
+		discord: 'Data',
+		},
+		RegistrationNew: {
+		_fallback: 'RegistrationTo198',
+		judgements: 'Vec<RegistrationJudgement>',
+		deposit: 'Balance',
+		info: 'IdentityInfoNew'
+		},
+		Username: 'Vec<u8>',
+	});
+
+	// We want to migrate `IdentityOf` storage item
+	let IdentityOfkeyToMigrate = "IdentityOf";
+	let IdentityOfHexkeyToMigrate = xxhashAsHex(IdentityOfkeyToMigrate, 128);
+
+	// We want to migrate `SubsOf` storage item
+	let SubsOfkeyToMigrate = "SubsOf";
+	let SubsOfHexkeyToMigrate = xxhashAsHex(SubsOfkeyToMigrate, 128);
```

Comment:
We also need to migrate `SuperOf` and `Registrars`

Reply:
They are being migrated also. The `identityMigration.js` is used to modify those storage items that can not be migrated as they are from the Relay Chain.

Reply:
I run this command:
```
yarn migrate -c wss://..... -f people-rococo.json -p Identity -m ./identityMigration.js
```
Which migrate the whole `Identity` pallet and apply modifications if necessary based on what's in the `identityMigration.js` file